metamorph/processor.go

```diff
@@ -171,6 +171,22 @@ func (p *Processor) Set(req *ProcessorRequest) error {
 	return p.store.Set(spanCtx, req.Hash[:], req.StoreData)
 }
 
+// Close all channels and goroutines for graceful shutdown
+func (p *Processor) Shutdown() {
+	p.logger.Infof("Shutting down processor")
+	p.processExpiredSeenTxsTicker.Stop()
+	p.processExpiredTxsTicker.Stop()
+	if p.ch != nil {
```

Comment:
where do we use `p.ch` 🤔  Don't remember what channel is that. I don't see it in the code, am I missing something?

Reply:
Also good point. Will check that.